install.sql

```diff
@@ -14,6 +14,7 @@ CREATE TABLE IF NOT EXISTS `%TABLE_PREFIX%newsmanager` (
     `status` tinyint(1) DEFAULT '0',
     `newsmanager_category_id` varchar(255) DEFAULT NULL,
     `title` varchar(255) NOT NULL DEFAULT '',
+    `subtitle` varchar(255) NOT NULL DEFAULT '',
```

Comment:
Generell vielleicht besser mittels `ensureColumn` in einer `install.php` arbeiten?
Siehe https://github.com/redaxo/docs/issues/25#issuecomment-335768649

Reply:
das wäre ein hinweis an den hauptentwickler [georgkaser](https://github.com/georgkaser). oder willst du?

Reply:
Ich will nicht, nein. Mir ist es nur aufgefallen, als es durch die Timeline gerauscht ist. Gerne Georg! :)

Reply:
Lese mit - bin aber noch nicht so weit mich darum zu kümmern. Gerne kann einer von euch auch so einen Admin Zugriff haben - wenn einer von euch mehr Zeit hat. Wie kann ich den Zugriff übertragen?

Reply:
@georgkaser Alle FOR-Mitglieder haben direkten Schreibzugriff auf alle Repos. Alle Nicht-FORs können PRs erstellen. Brauchst also nichts weiter zu tun.

Reply:
ich würde gerne, aber ich bin zu sehr laie als das ich das komplett übersetzen könnte…

rex_sql_table::get(rex::getTable('foo'))
    ->ensurePrimaryIdColumn()
    ->ensureColumn(new rex_sql_column('title', 'varchar(255)'))
    ->ensureColumn(new rex_sql_column('description', 'text', true))
    ->ensure();

wie hieße den die übersetzung für `title` varchar(255) NOT NULL DEFAULT '',

rex_sql_table::get(rex::getTable('%TABLE_PREFIX%newsmanager')) // ???
    ->ensurePrimaryIdColumn()
    ->ensureColumn(new rex_sql_column('title', 'varchar(255)'))
    ->ensureColumn(new rex_sql_column('NOT NULL', 'default', true)) // ???
    ->ensure();